configure.js

```diff
@@ -33,7 +33,13 @@ exports.init = function() {
         state.install = render(ctx, {})
         pluginParser.loadPlugins(function(data){
             // actions can only be bound to grid after full initialization
-            state.install.set("plugins", data).then(libplugin.bindDeleteButtons(state));
+            var temp_plugins;
+            temp_plugins = state.install.get("plugins");
+            if (temp_plugins == undefined) {
+                temp_plugins = [];
+            }
+            temp_plugins.push(data);
+            state.install.set("plugins", temp_plugins).then(libplugin.bindDeleteButtons(state));
```

Comment:
To be clear, the sense of my comment is that I'd like to get rid of caching vectors here and just propagate the change specific to that plugin. I'd rather do this because I think it leads to cleaner code.